pkg/controllers/vdb/podfacts.go

```diff
@@ -240,6 +240,18 @@ func MakePodFactsForSandbox(vrec config.ReconcilerInterface, prunner cmds.PodRun
 	return pf
 }
 
+// Clone will make a new copy of the podfacts, but setup for the given sandbox name.
+func (p *PodFacts) Clone(sandbox string) PodFacts {
```

Comment:
I would suggest calling this function `Copy` or `CopyForSandbox`. `Clone` gives the impression we are not going to change anything.

Reply:
Good point. I'll use `Copy`